### DISCUSSION TITLE
¿Puedo tener un campo numérico con el resultado de una sumatoria?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#76] [2. Questions and Answers] [answered]
Hola
Tengo las questions A,B,C y quiero que C tenga la sumatoria de A + B. Creo que no es posible que en el momento que ingrese A y B mi campo C me entrega el resultado y lo puedo llenar cuando envié el formulario en el campo vacío?.

¿Es posible realizar eso y cómo sería?

Tengo un caso pero no es de sumatoria se de división con multiplicación.

Atte.

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Te podría servir mucho la sección de Question Exec de la documentación:

https://doc.cotalker.com/docs/documentation/automation/question_exec?highlight=question~exec

Lamentablemente, todavía está como "work in progress", sin embargo, el issue #8 hace referencia a lo que posteriormente será la documentación de esta página. Si bien actualmente no existe un evento que dinámicamente escriba C a partir de cambios de valores de A y B, si puedes escribir en C, como una acción luego de presionar "enviar formulario" usando el evento Validate o Post Save, con la directiva SET_RESPONSES.

**Reply by NicolasPLG** [reply]
Entiendo @guillevalin pero tengo un problema, quizas no tenga los permisos pero cuando entro al link no veo documentación :(.

![image](https://user-images.githubusercontent.com/61419471/114627132-be75ba00-9c82-11eb-86ce-ddf6dba607a2.png)

Con respecto a lo que me indicas:  ¿me puedes mostrar en un ejemplo como se debe escribir?

Gracias

**Reply by guillevalin** [reply]
@NicolasPLG el issue #8 era un link, ahí escribí como se usa.

**Reply by acedward** [reply]
Está arriba el link original

**Reply by NicolasPLG** [reply]
OK lo leeré y voy a tratar de aplicarlo. gracias @acedward , @guillevalin 

Salu2

**Reply by acedward** [reply]
@guillevalin igual podría estar difícil llamar a SET_RESPONSES en el campo donde está la la "suma" de los valores. Quizás sería más fácil con un SET_RESPONSES con Identifier target

**Reply by NicolasPLG** [reply]
No entiendo mucho la documentación, tienen por casualidad un documento o link que tenga conceptos un poco más simples de utilizar. Busqué por la net y no sé si colocar la expresión $sum : identifier 1, identifier 2, sea lo correcto :(
